src/python/WMCore/Services/Rucio/RucioUtils.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+This module contains a few basic WMCore-related utilitarian
+Rucio data structures and functions
+"""
+
+import random
+
+RUCIO_VALID_PROJECT = ("Production", "RelVal", "Tier0", "Test", "User")
+# grouping values are extracted from:
+# https://github.com/rucio/rucio/blob/master/lib/rucio/common/schema/cms.py#L117
+GROUPING_DSET = "DATASET"
+GROUPING_ALL = "ALL"
+
+
+def validateMetaData(did, metaDict, logger):
+    """
+    This function can be extended in the future, for now it will only
+    validate the DID creation metadata, more specifically only the
+    "project" parameter
+    :param did: the DID that will be inserted
+    :param metaDict: a dictionary with all the DID metadata data to be inserted
+    :param logger: a logger object
+    :return: False if validation fails, otherwise True
+    """
+    if metaDict.get("project", "Production") in RUCIO_VALID_PROJECT:
+        return True
+    msg = "DID: %s has an invalid 'project' meta-data value: %s" % (did, metaDict['project'])
+    msg += "The supported 'project' values are: %s" % str(RUCIO_VALID_PROJECT)
+    logger.error(msg)
+    return False
+
+
+def weightedChoice(choices):
+    # from https://stackoverflow.com/questions/3679694/a-weighted-version-of-random-choice
+    # Python 3.6 includes something like this in the random library itself
+
+    total = sum(w for c, w in choices)
+    r = random.uniform(0, total)
+    upto = 0
+    for c, w in choices:
+        if upto + w >= r:
+            return c
+        upto += w
+    assert False, "Shouldn't get here"
+
+
+def isTapeRSE(rseName):
+    """
+    Given an RSE name, return True if it's a Tape RSE (rse_type=TAPE), otherwise False
+    :param rseName: string with the RSE name
+    :return: True or False
+    """
+    # NOTE: a more reliable - but more expensive - way to know that would be
+    # to query `get_rse` and evaluate the rse_type parameter
+    return rseName.endswith("_Tape")
```

Comment:
because you use this pattern at least twice and due to above comment I suggest to put `rseName.endswith()` into separate function which will check the rse name. Its implementation can be as simple as this return statement or more sophisticated as in a NOTE message.